### PR TITLE
[Bug] BraceFixer - fix no token before start.

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1957#issuecomment-223771898

opening now to see Travis results

### CODE REVIEW

**Comment by SpacePossum** [comment]
fabbot is wrong on this one.
Expecting HHVM to fail

**Comment by SpacePossum** [comment]
HHVM failures become more and more interesting :D

**Comment by keradus** [comment]
@SpacePossum 
are you sure that target should be 1.12 and not 1.11 ?

**Comment by SpacePossum** [comment]
>are you sure that target should be 1.12 and not 1.11 ?

I started out on a bug on 1.12, but I think this might indeed needs a retarget to 1.11. I'll finish it first :)

File: Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php [hunk 1]
```diff
@@ -63,6 +63,10 @@ public function fix(\SplFileInfo $file, $content)
     {
         $tokens = Tokens::fromCode($content);
 
+        if (count($tokens) < 1 || !$tokens[0]->isGivenKind(T_OPEN_TAG)) {
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php]: Do we really call fixer with empty tokens collection?

Comment (keradus) [inline Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php]: if so, we should put that check to `AbstractFixer::isCandidate`

Comment (SpacePossum) [inline Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php]: >AbstractFixer::isCandidate

agreed, will do on 2.x :)

Comment (keradus) [inline Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php]: but first, do we?
> Do we really call fixer with empty tokens collection?

**Comment by SpacePossum** [comment]
This PR fixes a bunch of issues including the one ref. in the description.
ping @GrahamCampbell as reporter of the stated issue.
All green now :)

**Comment by keradus** [comment]
so, I should merge it to 1.11 ?

**Comment by keradus** [comment]
nice fabbot failure btw...

**Comment by SpacePossum** [comment]
Yeah fabbot is not as smart as it could be about spelling ;)
Anyway, here is a commit for 1.11 https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/533ac2065fd7f33fb82a6d097e8861d8fe1e4165

we can ditch this PR than as don't need the typo fixes in the comments of `NativeFunctionCasingFixer`

**Pull request closed without merging**